model/user_autocomplete.go

```diff
@@ -17,6 +17,22 @@ type UserAutocompleteInTeam struct {
 	InTeam []*User `json:"in_team"`
 }
 
+type UserAutocomplete struct {
+	InSystem     []*User `json:"in_system"`
```

Comment:
Yep, that's correct. Then later we might extend it to have more fields as well but this is perfect for right now